### PR TITLE
Update schema-behaviors.html.md

### PR DESCRIPTION
**Pull request by mbfrahry**
After some discussion with @nmuesch and @lafentres,  I've drafted this PR to get rid of some confusion around values that are invalid for attributes.

### CODE REVIEW

**Comment by nfagerlund** [comment]
Hmm, so am I understanding right that the only valid uses are "present and `true`" or "absent"? If so, do you think that might want a little more explanation, or is it a familiar situation in Go plugins?

**Comment by mbfrahry** [comment]
Right, it's either true or you don't use it. We've got validation in place that complains if you use `Required: false` instead of `Optional: true`. I don't have the full context for why that is the way it is but it's tripping up provider developers. I'm open to hearing a better solution on representing this issue.

File: content/source/docs/extend/schemas/schema-behaviors.html.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ configuration, as opposed to trying to update the existing resource.
 ## Primitive Behaviors
```
Comment (nfagerlund) [inline content/source/docs/extend/schemas/schema-behaviors.html.md]:
```suggestion
## Primitive Behaviors 

-> **Note:** The primitive behavior fields cannot be set to `false`. You can opt out of a behavior by omitting it.

```

Comment (mbfrahry) [reply content/source/docs/extend/schemas/schema-behaviors.html.md]: Sounds good! Thanks!

**Pull request merged**